### DISCUSSION TITLE
custom logstash filter

### DISCUSSION THREAD

**Discussion by dubnik** [Security-Onion-Solutions/securityonion#8037] [Unsupported Versions]
Hi,

How can I please add custom logstash filter into SO. I would like to add new syslog listner on specific port and send logs from Fortinet to this port and store in ELK.

I followed the documentation and added custom .conf file into

`/opt/so/saltstack/local/salt/logstash/pipelines/config/custom/`

and updated

logstash:
  pipelines:
    manager:
      config:
        - so/0009_input_beats.conf      
        - so/0010_input_hhbeats.conf
        - so/0011_input_endgame.conf
        - so/9999_output_redis.conf.jinja
	- custom/1000_firewall_auth.conf

But logstash not nothing changes and nothing is listening on specified port. Or is there better way to achieve what  I need ?

Thanks

**Comment by dougburks** [comment]
Since you're sending logs from Fortinet, have you considered using the Fortinet filebeat module?
https://docs.securityonion.net/en/2.3/filebeat.html#example-2-fortinet-logs

**Reply by dubnik** [reply]
Hi 

I've also tried using filebeat fortinet module, but I have one issue that message is not parsed and I am not able to search values from message. All message is in one raw.

Is there any workaround to parse whole message?

Thanks

**Reply by dougburks** [reply]
Check the filebeat and elasticsearch log files for additional clues.
